src/components/MarketingBanner.vue

```diff
@@ -74,31 +73,9 @@
   flex-shrink-0
 }
 
-.marketing-banner__background-image {
```

Comment:
Removing these for now. Looks plain, but Karim is thinking through different motifs.